cmd/adapter-rest/migratecmd/migrate.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package migratecmd
+
+import (
+	"fmt"
+
+	cmdutils "github.com/trustbloc/edge-core/pkg/utils/cmd"
+
+	"github.com/spf13/cobra"
+	"github.com/trustbloc/edge-core/pkg/log"
+	"github.com/xo/dburl"
+
+	"github.com/trustbloc/edge-adapter/pkg/db/migrate"
+)
+
+const (
+	datasourceNameFlagName  = "dsn"
+	datasourceNameFlagUsage = "Datasource Name with credentials if required," +
+		" eg. mysql://root:secret@localhost:3306/adapter?parseTime=true"
+
+	migrationDirectionFlagName  = "dir"
+	migrationDirectionFlagUsage = "Direction in which to migrate the datasource schemas. Valid values: 'up', 'down'."
+)
+
+var logger = log.New("edge-adapter/migrate")
+
+type config struct {
+	dsn *dburl.URL
+	dir int
+}
+
+// GetMigrateCmd returns the cobra migrate command.
+func GetMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "DB migration.",
+		Long:  "Upgrades and downgrades SQL database schemas.",
```

Comment:
Just curious about the migration command - assume we have a DB running and multiple instances of adapter (for load balancing), from which instance are we gonna run the migration command? Are we gonna have one instance purely for sql db migration ?

@llorllale @troyronda @sudeshrshetty @fqutishat

Reply:
This PR allows the edge-adapter to migrate DB schemas via the CLI only.